plugin/META-INF/MANIFEST.MF

```diff
@@ -9,6 +9,7 @@ Automatic-Module-Name: amazon.q.eclipse
 Bundle-ActivationPolicy: lazy
 Bundle-Activator: software.aws.toolkits.eclipse.amazonq.plugin.Activator
 Require-Bundle: org.eclipse.core.runtime;bundle-version="3.31.0",
+ org.tukaani.xz,
```

Comment:
Why do we have an OSGI bundle dependency as well as a Maven dependency? We should only have one or the other. If this library is available through OSGI let's go that route (but also keep a version lock on it).

Keep in mind that we will need to add an entry in our attribution.xml for the new dependency as well: https://github.com/aws/amazon-q-eclipse/blob/main/attribution.xml